scripts/core/auth/login-modal.html

```diff
@@ -9,12 +10,12 @@
         <div class="form-handler" ng-show="!changePassword">
             <figure class="avatar" style="background-image:url('{{ identity.Avatar.href || 'images/avatar_200.png' }}')" ng-show="identity.Avatar"></figure>
             <form name="loginForm" ng-submit="authenticate()">
-                <fieldset class="inputs">
+                <fieldset class="inputs" ng-show="features.auth_db">
```

Comment:
Doesn't `ng-if` work here?

Reply:
no it doesn't, it's angular thing, the form it sends then is empty

Reply:
I think ng-if creates a new child scope for everything inside it, that's why